### PR TITLE
[feature](segcompaction) enable segcompaction by default

### PR DESCRIPTION
**Pull request by freemandealer**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by freemandealer** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]
LGTM

File: docs/zh-CN/docs/admin-manual/config/be-config.md [hunk 1]
```diff
@@ -637,7 +637,7 @@ Metrics: {"filtered_rows":0,"input_row_num":3346807,"input_rowsets_count":42,"in
 
 * 类型：bool
 * 描述：在导入时进行 segment compaction 来减少 segment 数量
-* 默认值：false
```
Comment (zhannngchen) [inline docs/zh-CN/docs/admin-manual/config/be-config.md:640]: please also update the document `data-faq.md` Q4

**Comment by yiguolei** [comment]
@freemandealer please submit to doris master directly

**Comment by freemandealer** [comment]
> @freemandealer please submit to doris master directly

@yiguolei  done #18722

**Pull request closed without merging**